conan-exilesconfig.json

```diff
@@ -161,7 +158,7 @@
         "IncludeInCommandLine":false,
         "DefaultValue":"0",
         "EnumValues":{
-            "0":"Europe",
+            "0":"Europe (default)",
```

Comment:
You knew I'd ask this. Is there a benefit to calling out a default when it is different for all users?

Reply:
Fixed